addons/mail/static/src/components/discuss_sidebar_mailbox/discuss_sidebar_mailbox.scss

```diff
@@ -1,57 +1,5 @@
-// ------------------------------------------------------------------
-// Layout
-// ------------------------------------------------------------------
-
 .o_DiscussSidebarMailbox {
-    display: flex;
-    align-items: center;
-    padding: map-get($spacers, 1) 0;
-}
-
-
-.o_DiscussSidebarMailbox_item {
-    margin-left: $o-mail-discuss-sidebar-category-item-margin;
-    margin-right: $o-mail-discuss-sidebar-category-item-margin;
-
-    &:first-child {
-        margin-left: $o-mail-discuss-sidebar-category-item-avatar-left-margin;
-    }
-
-    &:last-child {
-        margin-right: $o-mail-discuss-sidebar-scrollbar-width;
-    }
-}
-
-
-.o_DiscussSidebarMailbox_name {
-    @include text-truncate();
-}
-
-// ------------------------------------------------------------------
-// Style
-// ------------------------------------------------------------------
-
-.o_DiscussSidebarMailbox {
-    cursor: pointer;
-
-    &.o-active {
-        background-color: gray('200')
-    }
-
     &:hover {
-        background-color: gray('300');
+        background-color: gray('300') !important;
```

Comment:
I think you could use the same technique :
https://github.com/odoo/odoo/pull/89908#discussion_r861773499

Don't hesitate to ask if you have questions :)

Reply:
I answer to this remark here, and by extension, to your comment here https://github.com/odoo/odoo/pull/89908#discussion_r861773499 too 😉 

I like this solution, especially for the fact of allowing us to remove the css on `:hover` related to this component. However, this solution leads to some inconsistencies, such as `bg-colour:hover !== bg-300` or being sure that in the XML this element it is 100% a button and not something else. This point is just a little detail, but in the big picture it works fine.

What do you think about this @odoo/rd-discuss ?

Reply:
Alright! I'll adapt this `<button/>` logic here and in `DiscussSidebarCategoryItem`.

Thanks a lot guys! 👍